### ISSUE TITLE
converted weights do not reproduce the results

### ISSUE THREAD

**Issue by arghavan-kpm** [Separius/SimCLRv2-Pytorch#5]
Hello, when I get the output of verify.py for converted (pretrained) r50_1x_sk0, it is 45.09 with --ema and 45.76 without --ema. I checked the output of model (before fc) and they were all zero. I also checked other pretrained simclr models and the same thing happened. I tried with both tf==2.3.0 and tf==1.15.0.

**Comment by witold-oleszkiewicz** [comment]
Confirm, I have the same problem. There is also a bug in accuracy evaluation, so in fact  the accuracy is less than 1% (top-1).

**Comment by zhangzilongc** [comment]
I have the same condition as you
